packages/@react-spectrum/menu/docs/Menu.mdx

```diff
@@ -0,0 +1,357 @@
+
+  return (
+    <>
+      <MenuTrigger>
+        <ActionButton>
+          Edit
+        </ActionButton>
+        <Menu onAction={setAction}>
+          <Item key="cut">Cut</Item>
+          <Item key="copy">Copy</Item>
+          <Item key="paste">Paste</Item>
+        </Menu>
+      </MenuTrigger>
+      <p>Action: {action}</p>
+    </>
+  );
+}
+```
+
+## Selection
+Menu supports multiple selection modes. By default, selection is disabled, however this can be changed using the `selectionMode` prop.
+Use `defaultSelectedKeys` to provide a default set of selected items (uncontrolled) and `selectedKeys` to set the selected items (controlled). The value of the selected keys must match the `key` prop of the items.
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+    Align
+  </ActionButton>
+  <Menu selectionMode="single" defaultSelectedKeys={['middle']}>
+    <Item key="left">Left</Item>
+    <Item key="middle">Middle</Item>
+    <Item key="right">Right</Item>
+  </Menu>
+</MenuTrigger>
+```
+The `selectedKeys` prop can be used to make the selected state controlled.
+
+```tsx example
+function Example() {
+  let [selected, setSelected] = React.useState(new Set(['middle']));
+
+  return (
+    <>
+      <MenuTrigger>
+        <ActionButton>
+          Align
+        </ActionButton>
+        <Menu selectionMode="single" selectedKeys={selected} onSelectionChange={setSelected}>
+          <Item key="left">Left</Item>
+          <Item key="middle">Middle</Item>
+          <Item key="right">Right</Item>
+        </Menu>
+      </MenuTrigger>
+      <p>Current selection (controlled): {[...selected]}</p>
+    </>
+  );
+}
+```
+
+Set `selectionMode` prop to `multiple` to allow more than one selection.
+
+```tsx example
+function Example() {
+  let [selected, setSelected] = React.useState(new Set(['Sidebar', 'Console']));
+
+  return (
+    <>
+      <MenuTrigger closeOnSelect={false}>
+        <ActionButton>
+          Show
+        </ActionButton>
+        <Menu selectionMode="multiple" selectedKeys={selected} onSelectionChange={setSelected}>
+          <Item key='Sidebar'>Sidebar</Item>
+          <Item key='Searchbar'>Searchbar</Item>
+          <Item key='Tools'>Tools</Item>
+          <Item key='Console'>Console</Item>
+        </Menu>
+      </MenuTrigger>
+      <p>Current selection (controlled): {[...selected].join(', ')}</p>
+    </>
+  );
+}
+```
+
+## Complex Menu Items
+
+Items within Menu also allow for additional content used to better communicate options. Icons and descriptions can be added to the `children` of `Item` as shown in the example below. If a description is added, the prop `slot="description"` must be used to distinguish the different `<Text>` elements.
+
+[View guidelines](https://spectrum.adobe.com/page/popovers/#Content-types)
+
+```tsx example
+import {Keyboard, Text} from '@react-spectrum/text';
+<MenuTrigger>
+  <ActionButton>
+    Edit
+  </ActionButton>
+  <Menu>
+    <Item key="cut" textValue="cut">
+      <Cut />
```

Comment:
Hmm we should really do this by default, but I think you need `size="S"` here for it to look right?

Reply:
Oh whoops I may have deleted it.